.travis.yml

```diff
@@ -1,16 +1,57 @@
 sudo: false
+env:
+  - DJANGO_DEBUG=True DJANGO_SETTINGS_MODULE=memopol.settings
 language: python
 python:
-  - "2.7"
+- '2.7'
+before_install:
+- pip install codecov
 install:
-  - pip install django
-  - pip install -U pip
-  - pip install -r requirements.txt
-  - cp memopol/config.json.sample memopol/config.json
+- pip install -e .
+- pip install flake8 pep8 pytest pytest-django pytest-cov codecov
```

Comment:
would be nice to have this in a test-requirements.txt file (similar to requirements.txt)